src/App.jsx

```diff
@@ -36,36 +40,70 @@ function App() {
     setMoves((currentMoves) => currentMoves + 1);
   };
 
+  let rgb = [...rgbStart];
+  if (moves <= 60) {
+    rgb[0] += 3 * moves;
+    rgb[1] -= 3 * moves;
+    rgb[2] -= moves;
+  } else {
+    rgb = [...rgbEnd];
+  }
```

Comment:
What is this doing?

Reply:
It changes the color of the moves counter based on the moves. Starts out as green and slowly turns read as the moves increases.
You can checkout the video to see how it's changing colors.

Reply:
Ah - it's changing the move counter! I was trying to figure out how it changed the tiles and thought I was going crazy.

One day I'll learn how to read.